x-pack/plugins/aiops/public/components/change_point_detection/split_field_selector.tsx

```diff
@@ -18,19 +18,34 @@ interface SplitFieldSelectorProps {
 export const SplitFieldSelector: FC<SplitFieldSelectorProps> = React.memo(({ value, onChange }) => {
   const { splitFieldsOptions } = useChangePointDetectionContext();
 
-  const options = useMemo<EuiSelectOption[]>(() => {
-    return splitFieldsOptions.map((v) => ({ value: v.name, text: v.displayName }));
+  const options = useMemo<EuiComboBoxOptionOption[]>(() => {
+    return splitFieldsOptions.map((v) => ({ value: v.name, label: v.displayName }));
   }, [splitFieldsOptions]);
 
+  const selection = options.filter((v) => v.value === value);
+
+  const onChangeCallback = useCallback(
+    (selectedOptions: EuiComboBoxOptionOption[]) => {
+      const option = selectedOptions[0];
+      if (typeof option !== 'undefined') {
+        onChange(option.label);
+      }
+    },
+    [onChange]
+  );
+
   return (
     <EuiFormRow>
-      <EuiSelect
+      <EuiComboBox
```

Comment:
Not introduced by this PR, but I guess the change point agg works if there is no split field? Currently for a data view with a metric field but not keyword field (such as the `event_rate` data set) the page errors when trying to set the full time range and run the analysis:

<img width="1212" alt="image" src="https://user-images.githubusercontent.com/7405507/217232641-f9f79291-b949-47ff-8042-f770437cd02c.png">

I added this as a follow-up in https://github.com/elastic/kibana/issues/145703.

Reply:
Added single metric support in 61d2e6d8e042a6095f2bd04640277e994961066b. 
The issue on the screenshot is related to the time bounds. For some reason, with this data set "Use full data" control tries to set an invalid time range. I'll investigate in a follow-up.

Reply:
I created https://github.com/elastic/kibana/issues/150465 for the issue with the Use full data button.